tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'managed',
+				text: 'Test Text',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('put').withArgs('/terms_of_services/1234', expectedTermsOfServiceParam)
+				.returns(Promise.resolve(response));
+			termsOfService.update(TERMS_OF_SERVICE_ID, 'enabled', 'Test Text');
+		});
+
+		it('should call callback with the updated terms of service info when a 200 response is returned', function(done) {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'Test Text',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'put').yieldsAsync(null, response);
+			termsOfService.update(TERMS_OF_SERVICE_ID, 'enabled', 'Test Text', function(err, termsOfServiceObject) {
+				assert.ifError(err);
+				assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				done();
+			});
+		});
+
+		it('should return a promise resolving to updated terms of service object when a 200 response is returned', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'Test Text',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.stub(boxClientFake, 'put').returns(Promise.resolve(response));
+			return termsOfService.update(TERMS_OF_SERVICE_ID, 'enabled', 'Test Text')
+				.then(termsOfServiceObject => {
+					assert.strictEqual(termsOfServiceObject, response, 'terms of service object is returned');
+				});
+		});
+	});
+
+	describe('createUserStatuses()', function() {
+		it('should make POST request to create custom terms of service user status when called', function() {
+			var expectedParam = { body: { tos: {tos_id: '1234', type: 'terms_of_service'}, is_accepted: true} };
+
+			var response = {
+				type: 'terms_of_service_user_status',
+				id: '1234',
+				tos: {},
+				user: {},
+				is_accepted: true,
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('post').withArgs('/terms_of_service_user_statuses', expectedParam)
+				.returns(Promise.resolve(response));
+			termsOfService.createUserStatuses(TERMS_OF_SERVICE_ID, true, TERMS_OF_SERVICE_USER_ID);
```

Comment:
You're passing in a user ID here, but I don't see that reflected in the expected body params.  Make sure that this is being correctly passed through!